yadro/watcher.cpp

```diff
@@ -128,6 +124,78 @@ void dbuswatcher::updateSensors(void)
         }
     }
 }
+/**
+ * @brief update BMC version
+ */
+void dbuswatcher::updateBMCVersion()
+{
+    using objects = std::vector<sdbusplus::message::object_path>;
+
+    auto ep = getProperty<objects>(
+            sdbusplus::helper::OBJECT_MAPPER_IFACE,
+            "/xyz/openbmc_project/software/active",
+            "org.openbmc.Association",
+            "endpoints");
+
+    std::string ver;
+
+    if (!ep.empty())
+    {
```

Comment:
I believe the Coding Style prescribes having the opening curly on the same line with `if`.
Please run your file through `astyle` or `clang` with OpenBMC suggested arguments as given [here](https://github.com/openbmc/docs/blob/master/cpp-style-and-conventions.md#coding-style).

Reply:
In this case, done exactly as it says there.
However using `clang-format` is good idea and I would it later.
But why not now?.. :)